docs/self-managed/setup/guides/installing-payment-example.md

```diff
@@ -0,0 +1,13 @@
+---
+id: installing-payment-app-example
+title: "Verifying Camunda 8 installation with the payment example process application"
```

Comment:
This title is a little lengthy for a sidebar. We can keep as-is, but maybe add a sidebar_label named something like "Verifying Camunda 8 installation"?

Reply:
What about `Verifying Camunda 8 installation with an demo app`?